<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testsim</class>
 <widget class="QMainWindow" name="testsim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testsim</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial 8n1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="cmbBaud">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>22</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbPort">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>52</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>60</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Portname</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpen">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+O</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Simulation</string>
    </property>
    <widget class="QLineEdit" name="txtData_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>0000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkData_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data 1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkData_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data 2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtData_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkData_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data 3</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtData_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkData_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>82</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data 4</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtData_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkData_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>82</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data 5</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtData_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>206</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
